src/replication.rs

```diff
@@ -214,6 +453,8 @@ pub mod tests {
             storage_base_path: "/tmp".to_owned(),
             storage_threshold: 5000,
             replication_batch_size: 2,
+            backoff_min_wait: 5,
```

Comment:
Good catch, and  `backoff_min_wait` should be less than `backoff_max_wait`. For the test it doesn't really matter, because I'm only checking that a minimum waiting period occurs.